cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/AppMetadataStoreTest.java

```diff
@@ -134,9 +130,8 @@ public void apply() throws Exception {
     });
   }
 
-  @Ignore // Newly added; still debugging
```

Comment:
Are we going to un-ignore this test in this PR? Looks like it needs more changes. Maybe you can do it in a separate PR.